sklearn/utils/tests/test_validation.py

```diff
@@ -326,8 +330,9 @@ def test_check_array_min_samples_and_features_messages():
 
     # But this works if the input data is forced to look like a 2 array with
     # one sample and one feature:
-    X_checked = check_array(42, ensure_2d=True)
-    assert_array_equal(np.array([[42]]), X_checked)
+    # This might no loner be needed
```

Comment:
leave it there but catch the warning. We'll remove it once we change the behavior of ensure_2d.